static/img/kits/demand-and-capacity-management/demand-and-capacity-management-kit-logo.drawio.svg

```diff
@@ -0,0 +1,24 @@
+        <rect x="99.6" y="71.95" width="15" height="4" fill="rgb(255, 255, 255)" stroke="none" transform="rotate(-30,107.1,73.95)" pointer-events="none"/>
+        <rect x="112.14" y="76.95" width="24" height="4" fill="rgb(255, 255, 255)" stroke="none" transform="rotate(-30,124.14,78.95)" pointer-events="none"/>
+        <rect x="124.6" y="82.45" width="33" height="4" fill="rgb(255, 255, 255)" stroke="none" transform="rotate(-30,141.1,84.45)" pointer-events="none"/>
+        <rect x="85.6" y="61.45" width="27" height="4" fill="rgb(255, 255, 255)" stroke="none" transform="rotate(-30,99.1,63.45)" pointer-events="none"/>
+        <image x="115.1" y="49.06" width="60" height="36.54" xlink:href="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAARwAAACtCAMAAABVyzC2AAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAABjUExURQAAAP///////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////5pFDaAAAAAhdFJOUwAwaI8oWIe//yAYSIDvUAhAeK/fYPfPcLef1xDnOMeXp7OPSc8AAAAJcEhZcwAAFxEAABcRAcom8z8AAAVGSURBVHhe7Z1te6owDIarqBWpUl/wDdT9/195EEPpbGBsc0egz/1pWz7tudo0CWkqnhiNA/oJ2EymMznPCel38GARKbm8C3NnRX8EItbrzZZkeYB9ZQhIkgpJFiB2JEnFkixAjEgSixGZgEhIkoodWYAoju9PwCMbXI+8JwsQU5KkIonJBCYkiQU8ssGExoYDWYA4kiQVJ7IAMSNJKs5kAWJFklikZAIhKWKhyQRERpJUIAw0bEiSCoSBBkWSWKAaWMJ4ZOSeJSkpYoFIx+Am5hlZAOd0kF6VaFLEYk0mwOSeOMwNbqQzn5AJjEkRC0UmwKRXyMwNn795FlzIBNyaznxGJsAUkrdkAuJMklig26KECZKPZALMeYUUwnAlRSw2ZAIHUsRmQTbvWZAgNigllzD5VYJqKcHULXBgGfakiA1iHYKps88zdKMQ7sc9+GQDd5onSM4fxNzS2SLYeeC2B+Zc4XYKFm7bbc6NrL7DeR10wZVwsc58HpHVcy4kx2eSKZk950Z6PIGddWfBHec58Mp33CtGD45oosxhSoIFe3wgFmLidhU8SNB5IUREYrhcsXjEmrRgCOB5PkgKhqXyPRGN65xywcnzKkZYE+0QV787cS9sfl6Reb273MuNz1wP/n64ierCHYuz8vVsHzX7HWIb+NluEPLFHYfl5uChQKnb9l/HcrP27YCPT/S/tyI5qpVXPppppGwmO6rImxQjauWWn9j6sslS9mNWC84bFQ3/pL8wnaatkae1HvY2O7QICJtYytl4uLexf7y3bLZHNR1mQDRpqPF8i/2HioZ34IezX24ui0TeDgNzRfHuN66ZQX6o1YDOfP2q3WWxlbODHoZG4ewnYWEb9vlCUpHuuUPSAXN37YUkUsqZUivdU6FG6sXup5YsV0optdO92nnhofEbxd9wlnKTSxX1YFEtprf/tYA48u13ugvV3cjyzQIRZ3lU626qlK6CliXVvyaTwbqDOy7W6mno9BuRwa57q+jTuPJ3I4NV9zKUcKpkV9bQPuhiqSTW46Abiyi7dbSUlOr1rf33nT+js/rkhJH6ePdplt06HWCPpur4Vle033X8ikusD2/cZ71onJlEavOmfXbqSRU71OvgDWe+7FOR/7JSN/lft9qmf2WiiZ4qJeVf1Rk/cetv095F67G6L6e/Kxgls0F8+8iVyldUvqRe7J6ywd2wi7XWh2JRvcBJnYb8DT9XKt99v1hRw1s8LBe9UoH8fug06MXzzChSp29tuMyPJiuL0S5orZCfd3u1aimQp6NM0h0zTcrF29urrfQ5+3sBMd19eYot+5SLvprLV40zXqsjFl+0pSV+j+uId42RtOfqiLixY9h3dUSqGm5nJt6/W7VouAfkt1cuaLjpsu15O+ILSOsXz9nXWNmi/m6vxGS7hturGKiZb63ajAuvnuUwA/kLMG70Tt34LYwbvTOtCQgxIPtOVKPOmOx+U6MOJs8XaF4dvHdRUDO7DUlWAX9mYek84OMdjPx7wJaX8fDZA3ayHR7AJdjJdr7XTA3cOzJwySXMgOwEGRbBvXeB0eElzKMFVzKBlEkjUE4uYYIdPF5VwuRYqCYb3O/oeLDdwKRYcDolTCAIp1MSu+cVkk+D23qKQrvBdTrIzA2M04FHLmGcjvfNTBVubwFyT4ObQeDVKoObmSMxN7jpFY4rA/NeO9q8DKSIBb5eGdzEHGe5wU0g0AJncM9yRRbAvOJwIgsQY5KkAuIY3EBHkgUweTnEMYQkScWWLADiNBGTJBZkAVz+QAbA5Q9kAFg5AAAAAAAgR4h/hXHlLYGc3nQAAAAASUVORK5CYII=" preserveAspectRatio="none" pointer-events="none"/>
```

Comment:
This SVG contains embedded PNG Image data, that by its nature does not scale, making it no true vector graphics.
Also this specific file seems to have been generated differently from the rest.

Would you kindly ensure consistency over all logos and gallery drawing, please.

Reply:
Good point. I guess some logos were exported as png from figma where the kit icons were originally created. Will have a look at it.

Reply:
closing this thread, as I saw that @Grand-Thibault fixed it.